tasks.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This file should be inside the `.vscode` directory

Reply:
I'm not using visual studio (pure vim), so therefore do not have .vscode directory. Any better folder to place this in, or should I have not committed this file?

Reply:
Oh, if you're not using Visual Studio Code, you don't need this. This is like a per-project `.vimrc` but for VSCode.

Reply:
oh wow. thanks!